From eb0d7069afdc7583487694325b17a1b792eb0ec9 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Wed, 24 Jul 2013 17:57:44 +0800
Subject: [PATCH 019/115] ENGR00215174 disable java proguard function to avoid
 optimization.

disable java proguard function to avoid optimization.
So, the jni can use java class variable.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 WfdSink/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WfdSink/Android.mk b/WfdSink/Android.mk
index 1c65982..f7aba9f 100644
--- a/WfdSink/Android.mk
+++ b/WfdSink/Android.mk
@@ -9,6 +9,7 @@ LOCAL_PACKAGE_NAME := WfdSink
 LOCAL_CERTIFICATE := platform
 
 LOCAL_STATIC_JAVA_LIBRARIES := fsl.imx
+LOCAL_PROGUARD_ENABLED := disabled
 
 include $(BUILD_PACKAGE)
 
-- 
1.8.0

